modules/assisted-installer-booting-with-a-usb-drive.adoc

```diff
@@ -0,0 +1,17 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_content-type: PROCEDURE
+[id="booting-with-a-usb-drive_{context}"]
+= Booting with a USB drive
+
+To register nodes with the {ai-full} using a bootable USB drive, use the following procedure:
```

Comment:
In this introductory sentence, you use a colon at the end. Further down the page, you do not. I don't think this colon is needed. This was my fault, as I suggested in the previous review. I see now that it's the introductory sentence. 

Apologies.

Reply:
Fixed.